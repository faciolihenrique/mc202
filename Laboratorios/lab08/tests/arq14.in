#Turmas E e F em ordem quase aleatória
i 157642 Alex Wei
i 154815 Breno Pedroni
i 157743 Bruno Mendes Richau
i 160138 Caio Cesar de Campos Silva
i 157754 Caio Salvador Rohwedder
i 157809 Danilo Marcolino Valente
i 043473 Fernanda Elena de Barros Reis
i 104923 Fernando Henrique de Moura Bastos
i 159708 Gabriel Bertini de Oliveira
i 155477 Gabriel Souza Franco
i 160160 Guilherme Furlan
i 146328 Guilherme Victor Cunha Câmara
i 157986 Henrique Noronha Facioli
i 155742 Hugo Ralf Pereira
i 155758 Igor Gustavo Hitzschky Lema
i 155925 Joao Pedro Brasao Toledo
i 155951 Joao Vitor Buscatto Silva
i 155976 Jose Henrique Ferreira Pinto
i 156059 Julianny Favinha Donda
i 156074 Kaique de Souza Monteiro
i 057092 Kaiser Dias Schwarcz
i 108677 Khevin Iam Pettri Mituti
i 156131 Lais Tiemi Murakami
i 136454 Larissa Dominique Garcia
i 156175 Lauro Cruz e Souza
i 117706 Lucas Cavalcante de Barros
i 136625 Lucas Felix Dantas Rocha
i 156405 Lucas Rodolfo de Castro Moura
i 103179 Lucas Seiji Mizuta
i 156412 Lucas Silva Schanner
i 148804 Marcelo Miyazaki
i 105351 Maria Angelica de Araujo Barnabe
i 149332 Mateus Roque Buck
i 150770 Nicolas Hamacher Liepkaln
i 103771 Pedro Henrique da Silva
i 158354 Rafael Ferigolo da Cruz Silva
i 157065 Rafael Marques Miorim
i 157104 Raphael Mobis Tacla
i 158372 Raphael Pavan Silva
i 092844 Renato Aparecido de Oliveira
i 118596 Rodolfo Martins Perin
i 157198 Rodrigo Dahlstrom Hilkner
i 160245 Rogerio Guimaraes Meirelles
i 157228 Ronaldo Prata Amorim
i 157240 Sabrina Beck Angelini
i 157324 Tamara Martinelli de Campos
i 157413 Thiago Lugli Goncales
i 137796 Victor Otávio Andrade das Neves
i 151305 Vinicius Guilherme de Souza Maion
i 139240 Vítor Monteiro Francisco dos Santos
i 158526 Yuri Catarino Delgado
n
a
p
d
l
x
