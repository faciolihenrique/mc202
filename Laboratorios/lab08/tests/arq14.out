Ação: #Turmas E e F em ordem quase aleatória
Ação: i 157642 Alex Wei
Ação: i 154815 Breno Pedroni
Ação: i 157743 Bruno Mendes Richau
Ação: i 160138 Caio Cesar de Campos Silva
Ação: i 157754 Caio Salvador Rohwedder
Ação: i 157809 Danilo Marcolino Valente
